packages/mobile/src/features/videoPlayer/commponents/YouTubePlayer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Platform } from "react-native";
+import YouTube from "react-native-youtube";
+import { IPlayer } from "../models/IPlayer";
+import Global from '../../common/models/Global';
+
+
+export const YouTubePlayer = (props:IPlayer) =>{
+
+    const height = (Platform.OS=='ios')?"100%":0;
```

Comment:
można to lepiej sformatować jakieś spacje dodać bo zbite strasznie

Reply:
ok